config/importmap.rb

```diff
@@ -33,4 +33,6 @@
     #   pin file, to: "./readable_model.js"
     pin "/assets/blocks/#{name}", to: file, preload: true
   end
-end
\ No newline at end of file
+end
+pin "tailwindcss-safe-area" # @0.5.1
+pin "tailwindcss/plugin", to: "tailwindcss--plugin.js" # @3.4.4
```

Comment:
Whoa, @jasonpaulso, I never considered that we could use tailwind plugins if we installed them directly. This is great! I had mistakenly concluded that we'd have to switch the whole project to use node/npm in order to support those.

Reply:
Do you know what "tailwindcss/plugin" is doing as opposed to "tailwindcss-safe-area" ? Is this some kind of generalized plugin loading script? It's hard for me to tell what it's doing from the minified JS.

Reply:
Actually, my celebration may have been premature. :) I pulled down this branch to run it locally and I'm getting an import issue:

```
tailwind | Rebuilding...
tailwind | Error: Cannot find module 'tailwindcss-safe-area'
tailwind | Require stack:
tailwind | - /Users/keithschacht/repos/hostedgpt/config/tailwind.config.js
tailwind |     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
tailwind |     at Function._resolveFilename (pkg/prelude/bootstrap.js:1955:46)
tailwind |     at Function.resolve (node:internal/modules/cjs/helpers:108:19)
tailwind |     at Function.resolve (/snapshot/tailwindcss/standalone-cli/patch-require.js:34:38)
tailwind |     at _resolve (/snapshot/tailwindcss/node_modules/jiti/dist/jiti.js:1:251148)
tailwind |     at jiti (/snapshot/tailwindcss/node_modules/jiti/dist/jiti.js:1:253746)
tailwind |     at /Users/keithschacht/repos/hostedgpt/config/tailwind.config.js:61:5
tailwind |     at evalModule (/snapshot/tailwindcss/node_modules/jiti/dist/jiti.js:1:256443)
tailwind |     at jiti (/snapshot/tailwindcss/node_modules/jiti/dist/jiti.js:1:254371)
tailwind |     at /snapshot/tailwindcss/lib/lib/load-config.js:52:30 {
tailwind |   code: 'MODULE_NOT_FOUND',
tailwind |   requireStack: [ '/Users/keithschacht/repos/hostedgpt/config/tailwind.config.js' ]
tailwind | }
```

Was it working for you? I'm digging into how this is implemented...

Reply:
It _was_ definitely working for me. However, I'd not run the repo since I made this PR and now *nothing* is working... let me do some digging of my own as well

Reply:
can you confirm that the .js files are the in vendor directory in your local repo?

Reply:
Yes they are in there.

Reply:
The second answer on this page:
https://stackoverflow.com/questions/71640507/how-to-import-tailwind-plugin-in-rails-7

Suggests that there is a second part of tailwind.config that needs to be edited. I'm not sure if that's what the issue is.